src/runtime/pydict.cs

```diff
@@ -139,12 +139,13 @@ public PyObject Values()
         /// </remarks>
         public PyObject Items()
         {
-            IntPtr items = Runtime.PyDict_Items(obj);
-            if (items == IntPtr.Zero)
+            using var items = Runtime.PyDict_Items(this.obj);
```

Comment:
After https://github.com/dotnet/roslyn-analyzers/issues/3305 is fixed we can configure build to fail here if `using` would be omitted